### PR TITLE
Update filter

### PR DESCRIPTION
**Pull request by dr-arrgghh**

### CODE REVIEW

**Comment by dr-arrgghh** [comment]
Updated filter to filter by tags.

**Review by Piyotato — APPROVED** [review]
LGTM, but I haven't tested it directly.

Small note: should we discuss the syntax for the filter command to specify how the user wants to filter the entries? I think there is probably more to filter than just tags.

File: src/main/java/seedu/address/logic/parser/FilterCommandParser.java [hunk 1]
```diff
@@ -31,6 +31,6 @@ public FilterCommand parse(String args) throws ParseException {
```
Comment (comicalromance) [inline src/main/java/seedu/address/logic/parser/FilterCommandParser.java:31]: The parameter passed to the exception in line 29 should be FilterCommand.MESSAGE_USAGE, not FindCommand.MESSAGE_USAGE

File: src/main/java/seedu/address/logic/commands/FilterCommand.java [hunk 2]
```diff
@@ -18,17 +19,17 @@ public class FilterCommand extends Command {
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
```
Comment (comicalromance) [inline src/main/java/seedu/address/logic/commands/FilterCommand.java:19]: Can also replace "filters all persons whose names" with "filters all persons whose tags" in line 17.

File: src/main/java/seedu/address/logic/parser/FilterCommandParser.java (hunk 1 above)
Comment (dr-arrgghh) [reply src/main/java/seedu/address/logic/parser/FilterCommandParser.java:30]: Changes updated

File: src/main/java/seedu/address/logic/commands/FilterCommand.java (hunk 2 above)
Comment (dr-arrgghh) [reply src/main/java/seedu/address/logic/commands/FilterCommand.java:19]: Changes updated

**Pull request merged**